griptape/utils/prompt_stack.py

```diff
@@ -5,17 +5,22 @@
 
 if TYPE_CHECKING:
     from griptape.memory.structure import ConversationMemory
+    from griptape.processors.base_processors import BasePromptStackProcessor
 
 
 @define
 class PromptStack:
+    """A class representing a stack of prompts for a conversation."""
```

Comment:
Please rephrase with [this styleguide](https://google.github.io/styleguide/pyguide.html#384-classes) in mind.
> The class docstring should not repeat unnecessary information, such as that the class is a class.